embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/event/EmbraceEventServiceTest.kt

```diff
@@ -353,64 +353,41 @@ internal class EmbraceEventServiceTest {
         assertNull(eventService.getStartupMomentInfo())
     }
 
-    @Test
-    fun `verify find event ids using findEventIdsForSession()`() {
-        // Simulate the session moving forward in time, and having new Moments added and us retrieving the eventIds given the new time range
-        // Note that because of how the underlying cache works, if the size of the eventIds collection didn't change from the last time
-        // this method was invoked, the previously cached value will be returned. So while calling this method with arbitrary start/end
-        // times can return wrong values, how it is being used, that won't happen. This test will simulate the EXPECTED usage rather than
-        // the arbitrary usage.
-
-        val sessionBeginTime = 100L
-        fakeClock.setCurrentTime(sessionBeginTime)
-        eventService.startEvent("first")
-        fakeClock.setCurrentTime(fakeClock.now() + 1L)
-
-        // after a new Moment is logged and the time ticks forward, we should see it reflected in the cache
-        assertEquals(1, eventService.findEventIdsForSession(sessionBeginTime, fakeClock.now()).size)
-        fakeClock.setCurrentTime(fakeClock.now() + 50L)
-        eventService.startEvent("second")
-        fakeClock.setCurrentTime(fakeClock.now() + 1L)
-        eventService.startEvent("third")
-
-        // the new time range will only return 2 of the logged moments because the clock hasn't ticked forward
-        assertEquals(2, eventService.findEventIdsForSession(sessionBeginTime, fakeClock.now()).size)
-        fakeClock.setCurrentTime(fakeClock.now() + 1L)
-
-        // After the clock ticks forward, because of the caching, we will still only return 2. This is a perf optimization that will be
-        // OK in practice because we should only bust the cache if there's a new moment - this check is just to verify the caching works
-        // because it won't really happen in practice
-        assertEquals(2, eventService.findEventIdsForSession(sessionBeginTime, fakeClock.now()).size)
-
-        // After logging another moment, the cache is busted so after the clock ticks forward, we get all 4 moments
-        eventService.startEvent("fourth")
-        fakeClock.setCurrentTime(fakeClock.now() + 1L)
-        assertEquals(4, eventService.findEventIdsForSession(sessionBeginTime, fakeClock.now()).size)
-    }
-
     @Test
     fun `verify close clears the existing events`() {
```

Comment:
I feel like we could remove this test and just call close on the test below. Or maybe improve the naming on this one because "existing events" is a bit ambiguous.

Reply:
Fair enough - removed the old test case.